src/Uno.UI.Composition/Composition/CompositionAnimation.cs

```diff
@@ -53,20 +53,28 @@ public string Target
 		set => _target = value ?? throw new ArgumentException();
 	}
 
+#if __SKIA__
+	CompositionObject? _currentCompositionObject;
+#endif
+
 	internal virtual object? Start(ReadOnlySpan<char> propertyName, ReadOnlySpan<char> subPropertyName, CompositionObject compositionObject)
 	{
 #if __SKIA__
+		_currentCompositionObject = compositionObject;
 		Compositor.RegisterAnimation(this, compositionObject);
 #endif
 		return null;
 	}
 
 	internal virtual object? Evaluate() => null;
 
-	internal virtual void Stop()
+	internal virtual void Stop(CompositionObject compositionObject)
```

Comment:
... and I think that we could just use the method parameter instead of the field?